### PR TITLE
New package: dxatlas.CwSkimmer version 2.1

### LINKED ISSUE #113268: [Package Request]: CW Skimmer
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.dxatlas.com/Download.asp
Publisher: Afreet Software
Package Name: CW Skimmer
Description: Multi-channel CW decoder and analyzer.
Package Version: 2.1
Installer URL: https://www.dxatlas.com/CwSkimmer/Files/CwSkimmer.zip

### PR DESCRIPTION
**Pull request by Jvr2022**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Resolve #113268

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113292&drop=dogfoodAlpha

### CODE REVIEW

**Comment by Jvr2022** [comment]
fixing #113292

File: manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: dxatlas.CwSkimmer
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: dxatlas
+PackageName: CwSkimmer
+License: none
+ShortDescription: Multi-channel CW decoder and analyzer.]
```
Comment (AsciiWolf) [inline manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Multi-channel CW decoder and analyzer.

```

Typo.

**Comment by stephengillie** [comment]
Internet searches suggest CW is Morse code used by HAM Radio operators.

**Comment by AsciiWolf** [comment]
Yes, it indeed is. :-)

File: manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: dxatlas.CwSkimmer
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: dxatlas
+PackageName: CwSkimmer
+License: none
+ShortDescription: Multi-channel CW decoder and analyzer.
```
Comment (AsciiWolf) [inline manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]: We can probably change the description to something like:

```suggestion
ShortDescription: Multi-channel Morse code (CW) decoder and analyzer.

```

Comment (Jvr2022) [reply manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]: How do i change it

**Comment by Jvr2022** [comment]
How do I change that

File: manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]: Yeah, that's a little more clear 😃

Comment (AsciiWolf) [reply manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]: It should be possible to directly merge this change I sent. If not, you will have to make a new commit in your forked repo.

Comment (Jvr2022) [reply manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]: Need i click resolve

Comment (AsciiWolf) [reply manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]: Or no, my bad. Clicking resolve will probably just close this thread. There should be another button.

Comment (Jvr2022) [reply manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:10]: I open github in the browser i am now on mobile

**Comment by Jvr2022** [comment]
In the browser there is a commit button

File: manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: dxatlas.CwSkimmer
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: dxatlas
+PackageName: CwSkimmer
+License: none
+ShortDescription: Multi-channel Morse code (CW) decoder and analyzer.
+
+
```
Comment (AsciiWolf) [inline manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:12]: Oops, sorry, I have added two new lines in my previous suggestions. Please remove them:

```suggestion

```

Comment (Jvr2022) [reply manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml:12]: I do

**Comment by Jvr2022** [comment]
Can I get a review

**Comment by Jvr2022** [comment]
Can we add support

**Review by stephengillie — APPROVED** [review]

**Pull request merged**